client/core/voting_table.go

```diff
@@ -31,22 +31,25 @@ func NewVotingTable(voteType types.VoteType, voters types.Voters, majority Quoru
 	return &votingTable{
 		voteType: voteType,
 		voters:   voters,
-		votes:    types.Votes{},
+		votes:    types.NewVotes(),
 		quorum:   TwoThirdsPlusOneVoteQuorum,
 		majority: majority,
 	}, nil
 }
 
-func (table *votingTable) Add(vote types.AddressVote) error {
-	if !table.isVoter(vote.Address()) {
-		return fmt.Errorf("voter address not found in voting table: %#x", vote.Address().Hash())
+func (table *votingTable) Add(voteAddressed types.AddressVote) error {
```

Comment:
you could call it `addressVote`, maps better with `types.AddressVote` :)

Reply:
I'm logging the request in https://github.com/kowala-tech/kcoin/pull/613/files#diff-658294a439327f8305b9d63bf426c417R581